unwrap-smt/README.md

```diff
@@ -0,0 +1,123 @@
+# Debezium Unwrap SMT Demo
+
+This setup is going to demonstrate how to receive events from MySQL database and stream them down to PostreSQL database using [Debezium Event Flattening SMT](http://debezium.io/docs/configuration/event-flattening/).
+
+
+## Topology
+
+```
+                   +-------------+
+                   |             |
+                   |    MySQL    |
+                   |             |
+                   +------+------+
+                          |
+                          |
+                          |
+          +---------------v------------------+
+          |                                  |
+          |           Kafka Connect          |
+          |  (Debezium, JDBC, ElasticSearch  |
+          |           connectors)            |
+          |                                  |
+          +---+-----------+------------------+
+                          |
+                          |
+                          |
+                          |
+                  +-------v--------+
+                  |                |
+                  |   PostgreSQL   |
+                  |                |
+                  +----------------+
+
+
+```
+We are using a Docker Compose to deploy following components
+* MySQL
+* Kafka
+  * ZooKeeper
+  * Kafka Broker
+  * Kafka Connect with [Debezium](http://debezium.io/) and  [JDBC](https://github.com/confluentinc/kafka-connect-jdbc) Connectors
+* PostgreSQL
+
+## Usage
+How to run:
+
+```shell
+# Start the application
+DEBEZIUM_VERSION=0.6
```

Comment:
`export` missing?

Reply:
I do not think it is necessary but I've added it.